datadog_lambda/tracing.py

```diff
@@ -89,9 +91,30 @@ def _context_obj_to_headers(obj):
     }
 
 
-def extract_dd_trace_context(event):
+def get_dd_trace_data(low_headers, context):
```

Comment:
I don't think readers can really tell the differences between `get_dd_trace_data` and `extract_dd_trace_context` by just reading the function names. I think this suggests an actual problem and a better structure could look like this:

- extract_dd_trace_context(event, context)
  - extract_dd_trace_context_from_context(context)
  - extract_dd_trace_context_from_event(event)

Reply:
Good call